<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QWidget" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lMarca">
     <property name="text">
      <string>Marca</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leMarca"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lPrecio">
     <property name="text">
      <string>Precio</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lePrecio"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lCant_cuerdas">
     <property name="text">
      <string>Cantidad de cuerdas</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leCant_cuerdas"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lCant_teclas">
     <property name="text">
      <string>Cantidad de teclas</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="leCant_teclas"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lMetal_usado">
     <property name="text">
      <string>Metal usado</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="leMetal_usado"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pbStock">
     <property name="text">
      <string>Ver Stock</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pbAgregar">
     <property name="text">
      <string>Agregar Instrumento</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="CBtipoInstrumento">
     <item>
      <property name="text">
       <string>Guitarra</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Viento</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Teclados</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
